CHANGELOG.md

```diff
@@ -6,3 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Added
+
+-   GherLint CLI tool
+-   First lint rule `indentation`
+-   Add `--fix` cli option to fix lint errors (https://github.com/gherlint/gherlint/pull/31)
```

Comment:
```suggestion
-   Add `--fix` cli option to fix lint errors (https://github.com/gherlint/gherlint/pull/31)
```
Do we really need a repetitive `Add` keyword at the beginning, as all of these go under the `Added` heading?